x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportStartDataFrameAnalyticsAction.java

```diff
@@ -314,8 +326,10 @@ public void onFailure(Exception e) {
 
                 @Override
                 public void onTimeout(TimeValue timeout) {
-                    listener.onFailure(new ElasticsearchException("Starting data frame analytics [" + task.getParams().getId()
-                        + "] timed out after [" + timeout + "]"));
+                    auditor.error(
```

Comment:
I don't think we need to audit this. This will be returned to the user in the REST request. Also note that what really times out here is the waiting bit, the task may start successfully.